contracts/babylon/src/msg/contract.rs

```diff
@@ -32,6 +32,8 @@ pub struct InstantiateMsg {
     pub notify_cosmos_zone: bool,
     /// If set, this will instantiate a BTC light client contract
     pub btc_light_client_code_id: Option<u64>,
+    /// Encoded `Vec<BtcHeaderInfo>` in hex.
+    pub btc_light_client_initial_headers: String,
```

Comment:
Do we need first work and first height here?

Reply:
Nope, as you can see from the previous code, the first work and height are derived from the initial base header. This is consistent with the old logic.